plugins/source/xkcd/cloud-config-ui/src/form/index.tsx

```diff
@@ -49,18 +55,27 @@ export function Form({ initialValues }: Props) {
 
   return (
     <FormProvider {...formContext}>
-      <Stack spacing={2}>
-        <FormFieldGroup title={initialValues ? 'Update a source' : 'Create a source'}>
-          <Stack>
-            <Stack marginBottom={2}>
+      <Card>
+        <CardContent>
+          <Stack gap={2}>
+            <Box display="flex" justifyContent="space-between" alignItems="center">
+              <Typography variant="h5">Configure source</Typography>
+              <Box display="flex" justifyContent="space-between" alignItems="center" gap={1.5}>
+                <Logo src={`${assetPrefix}/images/xkcd.png`} alt="XKCD" />
+                <Typography variant="body1">XKCD</Typography>
+              </Box>
+            </Box>
+            <Stack>
               <Controller
                 control={control}
                 name="name"
                 render={({ field, fieldState }) => (
                   <TextField
                     error={!!fieldState.error}
                     fullWidth={true}
-                    helperText={fieldState.error?.message}
+                    helperText={
```

Comment:
@rhino1998 we have a new pattern (very recent, you didn't miss it) to use the `getFieldHelperText` util from the plugin-config-ui-lib that will keep showing the helper text in addition to the error text

Reply:
[a475603](https://github.com/cloudquery/cloudquery/pull/18837/commits/a4756037d7e34c4026289b600425903219e23a58)